person.rb

```diff
@@ -1,20 +1,27 @@
+require './corrector'
+
 class Person
   attr_accessor :name, :age, :permission
   attr_reader :id
 
-  def initialize(age, name = 'Unknown', permission: true)
+  def initialize(age, name, permission: true)
     @id = Random.rand(1..1000)
     @name = name
+    @corrector = Corrector.new(name)
     @age = age
     @permission = permission
   end
 
+  def validate_name
+    @name = corrector.correct_name
+  end
```

Comment:
- [ ] great job at implementing the validate_name method 👍  however you should be using the `@corrector` instance variable instead of the just `corrector`